pyaedt/misc/aedtlib_personalib_install.py

```diff
@@ -148,7 +148,7 @@ def install_toolkit(toolkit_dir, product, aedt_version):
         tool_dir = os.path.join(toolkit_dir, product, "PyAEDT")
     os.makedirs(lib_dir, exist_ok=True)
     os.makedirs(tool_dir, exist_ok=True)
-    files_to_copy = ["Console", "Run_PyAEDT_Script", "Jupyter"]
+    files_to_copy = ["Console", "Run_PyAEDT_Script", "Jupyter", "Run_Toolkit_Manager"]
     # Remove hard-coded version number from Python virtual environment path, and replace it with the corresponding AEDT
```

Comment:
```suggestion
    # Remove hard-coded version number from Python virtual environment path and replace it with the corresponding AEDT
```
No comma because there is only one subject (implied "you") here with two verbs. No comma is necessary.